### PR TITLE
Update past blogs to comply with new template standards

### PR DESCRIPTION
**Pull request by austin0**
Inserts the following after the author name in each blog:
```
:imagesdir: /
:url-prefix:
:url-about: /
```

Removes leading `/` on image tags for past blogs:
`image::/img/blog/200004-rest-stats-jconsole.png[align="center"]` -> `image::img/blog/200004-rest-stats-jconsole.png[align="center"]`

Updates older blogs to make use of relative linking/url prefixes:
`link:https://openliberty.io/guides/etc` -> `link:{url-prefix}/guides/etc`

Currently on draft and seems to be fine.

Signed-off-by: Austin Bailey <Austin.Bailey@ibm.com>

### CODE REVIEW

**Comment by lauracowen** [comment]
@steven1046 Would you mind reviewing this for us please? It looks ok to me but is there anything you can spot that might cause problems? Also, is there anything that jumps out to you as being unnecessary to do at a local level? I know we discussed setting some global variables but I don't think we concluded anything specific yet?

File: posts/2019-07-18-microprofile-30-developer-experience.adoc [hunk 1]
```diff
@@ -10,8 +10,9 @@ blog_description: "MicroProfile 3.0 provides improvements to the developer exper
...
 Laura Cowen <https://github.com/lauracowen>
-
-Welcome to MicroProfile 3.0 on Open Liberty! MicroProfile 3.0 provides improvements to the developer experience.  MicroProfile Rest Client provides simplified configuration of SSL authentication and multiple client interfaces, MicroProfile Health Check separates liveness and readiness endpoints so that Kubernetes can check for both, and MicroProfile Metrics provides multi-dimensional metrics and simpler counters. In addition, Open Liberty now supports the PostgreSQL relational database. Give it a try in link:/about/[Open Liberty] 19.0.0.7.
+:imagesdir: /
+:url-prefix:
+:url-about: /Welcome to MicroProfile 3.0 on Open Liberty! MicroProfile 3.0 provides improvements to the developer experience.  MicroProfile Rest Client provides simplified configuration of SSL authentication and multiple client interfaces, MicroProfile Health Check separates liveness and readiness endpoints so that Kubernetes can check for both, and MicroProfile Metrics provides multi-dimensional metrics and simpler counters. In addition, Open Liberty now supports the PostgreSQL relational database. Give it a try in link:/about/[Open Liberty] 19.0.0.7.
```
Comment (steven1046) [inline posts/2019-07-18-microprofile-30-developer-experience.adoc]: Does this need to be separated into two lines so `:url-about:` is `/` like the other posts?

Comment (austin0) [reply posts/2019-07-18-microprofile-30-developer-experience.adoc]: Yep, thanks for catching this.

**Comment by steven1046** [comment]
Don't we need the front matter in the other templates too: https://raw.githubusercontent.com/OpenLiberty/blogs/draft/templates/post-single-author.adoc and https://raw.githubusercontent.com/OpenLiberty/blogs/draft/templates/post-multiple-authors.adoc?

So that the right metadata gets copied into all new posts.

File: posts/2019-07-18-microprofile-30-developer-experience.adoc (hunk 1 above)
Comment (austin0) [reply posts/2019-07-18-microprofile-30-developer-experience.adoc]: Fixed and checked all the other posts for the same issue.

**Comment by austin0** [comment]
@steven1046 the templates have been updated already by @jakub-pomykala I think it's just waiting to be PR'd in.

**Comment by jakub-pomykala** [comment]
#782 should have all the templates updated

**Review by steven1046 — APPROVED** [review]

**Pull request merged**